libmodernize/main.py

```diff
@@ -144,4 +147,9 @@ def main(args=None):
         rt.summarize()
 
     # Return error status (0 if rt.errors is zero)
-    return int(bool(rt.errors))
+    return_code = int(bool(rt.errors))
+    # If we are enforcing python 3 compatibility, return a non-zero exit code if we had to modify
+    # any files.
+    if options.enforce and rt.files:
+        return_code |= 2
```

Comment:
Nice, so if there we had to change something under `--enforce` *and* there were errors, we get 3 as an exit code.